examples/2d/multiple_sprites_one_entity.rs

```diff
@@ -0,0 +1,95 @@
+//! Constructs a large entity composed of multiple sprites
+
+use bevy::input::common_conditions::input_pressed;
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(ImagePlugin::default_nearest())) // prevents blurry sprites
+        .add_systems(Startup, setup)
+        // the user can press the arrow keys to translate the parent entity in the window
+        .add_systems(
+            Update,
+            translate::<-5, 0>.run_if(input_pressed(KeyCode::ArrowLeft)),
+        )
+        .add_systems(
+            Update,
+            translate::<5, 0>.run_if(input_pressed(KeyCode::ArrowRight)),
+        )
+        .add_systems(
+            Update,
+            translate::<0, 5>.run_if(input_pressed(KeyCode::ArrowUp)),
+        )
+        .add_systems(
+            Update,
+            translate::<0, -5>.run_if(input_pressed(KeyCode::ArrowDown)),
+        )
+        .run();
+}
+
+#[derive(Component)]
+struct ParentSprite;
+
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    commands.spawn(Camera2dBundle::default());
+
+    // the entity is composed of four tiles arranged in a square
+    commands
+        .spawn((
+            ParentSprite,
+            SpatialBundle::default(), // the parent entity only needs a Transform component, for positioning
```

Comment:
The comment is not 100% correct: The `ParentSprite` needs all the components of the `SpatialBundle` in order to be rendered correctly as part of the hierarchy.

Reply:
Removed the comment